bundles/org.openhab.binding.bluetooth.bluegiga/src/main/java/org/openhab/binding/bluetooth/bluegiga/handler/BlueGigaBridgeHandler.java

```diff
@@ -584,7 +584,7 @@ public boolean bgFindPrimaryServices(int connectionHandle) {
                 .withConnection(connectionHandle)
                 .withStart(1)
                 .withEnd(65535)
-                .withUuid(UUID.fromString("00002800-0000-0000-0000-000000000000"))
+                .withUuid(UUID.fromString("00002800-0000-1000-8000-00805F9B34FB"))
```

Comment:
where do these numbers come from? are they random or defined somewhere?

Reply:
Bluetooth specification defines base UUID for standard GATT attributes, see https://www.bluetooth.com/specifications/assigned-numbers/service-discovery.

Reply:
Thanks. Sounds good.